index.html

```diff
@@ -10,10 +10,53 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
-
+    <header id="header">
+        <img id="logo" src="https://svgshare.com/i/shA.svg" alt="Karma Logo" width="50px" height="40px"/>
+        <nav id="navbar">
+            <ul>
+                <li><a id="dark-link" href="#">Meet Karma</a></li>
+                <li><a href="#">How it Works</a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <article id="article1">
+            <h1>Introducing Karma</h1>
+            <p>Bring WiFi with you, everywhere you go.</p>
+            <button id="button">Learn More</button>
+        </article>
+        
+        <article>
+            <h2>Everyone needs a little Karma.</h2>
+            <div id="article2">
+                <section class="section">
+                    <img class="section-image" src="https://svgshare.com/i/shC.svg" alt="Devices image"/>
+                    <h3>Internet for all devices</h3>
+                </section>
+                <section class="section">
+                    <img class="section-image" src="https://svgshare.com/i/sf3.svg" alt="Coffee image"/>
+                    <h3>Boost your productivity</h3>
+                </section>
+                <section class="section">
+                    <img class="section-image" src="https://svgshare.com/i/sf4.svg" alt="Refill image"/>
+                    <h3>Pay as You Go</h3>
```

Comment:
as we were discussing, `<h3>` here should be replaced with `<a>` or `<div>`. 
coz, this is not a section heading, this is just a text data. 
heading for this block is already up there: 
```html
<h2>Everyone needs a little Karma.</h2>
```